configs/nr-otel-collector-agent-linux.yaml

```diff
@@ -167,10 +167,6 @@ processors:
   batch:
 
   resource:
```

Comment:
```suggestion

```

Should we remove the resource processor from the config too?

Reply:
good 👁️  removed in da6c864d9aa3ce59beeff9187bc004af721028ec